dashboard/lib/tasks/seed_in_test.rake

```diff
@@ -1,3 +1,5 @@
 Rake::Task['db:test:prepare'].enhance do
+  ActiveRecord::Base.establish_connection('test')
   Rake::Task['seed:test'].invoke
+  ActiveRecord::Base.establish_connection(ENV['RAILS_ENV'])  
```

Comment:
(curiosity) Does this have the same behavior as (what I intend with)
```
existing_connection = ActiveRecord::Base.connection
ActiveRecord::Base.establish_connection('test')
Rake::Task['seed:test'].invoke
ActiveRecord::Base.establish_connection(existing_connection)
```
If not, does this behavior make sense?